opus/application/static_media/js/opus.js

```diff
@@ -127,30 +128,21 @@ var opus = {
             $(".op-reset-button button").prop("disabled", true);
         }
 
-        // compare selections and last selections
-        if (o_utils.areObjectsEqual(selections, opus.last_selections)) {
-            // selections have not changed from opus.last_selections
+        // compare selections and last selections, extras and last extras
+        if (o_utils.areObjectsEqual(selections, opus.lastSelections) && o_utils.areObjectsEqual(extras, opus.lastExtras)) {
+            // selections have not changed from opus.lastSelections
```

Comment:
This comment (meaning the one after the if) is still wrong. Although frankly I'm not sure there's any reason to have a comment here at all. The code is self-explanatory, and we should never have comments that just restate what the code says, because then they get out of date! Comments should only be for higher-level ideas that are not obvious to the reader.